xml/System.Data.Spatial/DbGeometry.xml

```diff
@@ -432,9 +432,9 @@
         <Parameter Name="index" Type="System.Int32" />
       </Parameters>
       <Docs>
-        <param name="index">The index.</param>
-        <summary>Returns an element of this DbGeometry value from a specific position, if it represents a geometry collection. &lt;param name="index"&gt;The position within this geometry value from which the element should be taken.&lt;/param&gt;&lt;returns&gt;The element in this geometry value at the specified position, if it represents a collection of other geometry values; otherwise <see langword="null" />.&lt;/returns&gt;</summary>
-        <returns>An element of this DbGeometry value from a specific position.</returns>
+        <param name="index">The position within this geometry value from which the element should be taken.</param>
+        <summary>Returns an element of this DbGeometry value from a specific position, if it represents a geometry collection.</summary>
+        <returns>The element in this geometry value at the specified position, if it represents a collection of other geometry values; otherwise <see langword="null" />.</returns>
```

Comment:
```suggestion
        <returns>The element in this geometry value at the specified position, if it represents a collection of other geometry values; otherwise, <see langword="null" />.</returns>
```